Models/DTO/Administrator/FilterPharmacyReportDTO.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace API.Models.DTO.Administrator
+{
+    public class FilterPharmacyReportDTO
+    {
+        [Required]
+        [JsonProperty("id")]
+        public int Id { get; set; }
```

Comment:
It is unclear which `id` is required here.

Reply:
I aggree.